Rubberduck.Parsing/Grammar/VBALexer.g4

```diff
@@ -302,6 +302,7 @@ WS : [ \t];
 GUIDLITERAL : '{' [0-9A-F]+ '-' [0-9A-F]+ '-' [0-9A-F]+ '-' [0-9A-F]+ '-' [0-9A-F]+ '}';
 IDENTIFIER :  ~[[\](){}\r\n\t.,'"|!@#$%^&*\-+:=; 0-9-/\\-] ~[[\](){}\r\n\t.,'"|!@#$%^&*\-+:=; -]*;
 LINE_CONTINUATION : [ \t]* UNDERSCORE [ \t]* '\r'? '\n';
+BARE_HEX_LITERAL : [0-9a-fA-F]+;
```

Comment:
I **really** don't like this lexer rule. A lot of it's matching space is already covered by `IDENTIFIER` (which is terrible). It  can't ever match hex literals starting with a letter, because those are already matched by the `IDENTIFIER` token.

Are we 100% sure we need this rule? And if yes, can we document what we need it for?